packages/mocaccino-dracut/definition.yaml

```
@@ -0,0 +1,6 @@
+name: mocaccino-dracut
+category: sys-kernel
+version: "1.0"
+requires:
+- name: dracut
+  category: sys-kernel
```

If the version is omitted, it implies that the package requires a package without version. To specify at least one, you need to use `version: ">=0`
```suggestion
  category: sys-kernel
  version: ">=0"
```

---

Re-thinking now. We don't have dracut as a standalone package, only in -extra. It's better to drop the requires entirely